js/src/d3-bundle.js

```diff
@@ -0,0 +1,42 @@
+import {
+  event,
+  select,
+  selectAll
+} from "d3-selection";
+
+import {
+  xhr,
+  json
+} from 'd3-xhr';
+
+import {
+  range,
+  sum,
+  ascending,
+  descending
+} from 'd3-arrays';
+
+import * as scale from 'd3-scale';
```

Comment:
'import' is only available in ES6 (use esnext option).<br>Mixed double and single quotes.